src/test/java/com/epam/tc/hw7/ReadTestDataFromJson.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.tc.hw7;
+
+import java.io.FileReader;
+import java.io.IOException;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+public class ReadTestDataFromJson {
+
+    public MetalsColoursData readJsonToMetalsColoursData(int j) {
+
+        String fileName = "src/test/resources/JDI_ex8_metalsColorsDataSet.json";
+        JSONObject jsonObject = null;
+        try {
+            jsonObject = (JSONObject) new JSONParser().parse(new FileReader(fileName));
```

Comment:
It's bad idea to use that. You call this method in cycle above. Need to change to single open file